### PR TITLE
🏗 Report and store the bundle size of alternate runtime and extension files

### PR DESCRIPTION
**Pull request by danielrozenberg**
Releated to https://github.com/ampproject/amp-github-apps/issues/271

### CODE REVIEW

File: build-system/tasks/bundle-size.js [hunk 1]
```diff
@@ -100,10 +109,7 @@ async function storeBundleSize() {
       json: true,
       body: {
         token: process.env.BUNDLE_SIZE_TOKEN,
-        // TODO(#20843, danielrozenberg): add extensions and other runtimes.
-        bundleSizes: {
```
Comment (rcebulko) [inline build-system/tasks/bundle-size.js]: Nice foresight designing this api/structure

File: build-system/tasks/bundle-size.js [hunk 2]
```diff
@@ -208,7 +213,7 @@ function getLocalBundleSize() {
       cyan(shortSha(gitCommitHash())) + '.'
     );
   }
-  getBrotliBundleSize();
+  getBrotliBundleSizes();
```
Comment (rcebulko) [inline build-system/tasks/bundle-size.js]: What's the point of this call? Should this function be returning something? JSDoc would help, but also not clear if there's any point to this function call

File: build-system/tasks/bundle-size.js (hunk 1 above)
Comment (danielrozenberg) [reply build-system/tasks/bundle-size.js]: Just following the steps that I detailed in https://github.com/ampproject/amp-github-apps/issues/271 :)

File: build-system/tasks/bundle-size.js (hunk 2 above)
Comment (danielrozenberg) [reply build-system/tasks/bundle-size.js]: `getBrotliBundleSizes` prints its own logs. @rsimha added the `getLocalBundleSize()` method originally, I'm just updating here

Comment (rcebulko) [reply build-system/tasks/bundle-size.js]: So the point of this is just to log stuff?

Comment (danielrozenberg) [reply build-system/tasks/bundle-size.js]: Yes, it's a debugging thing

Comment (rcebulko) [reply build-system/tasks/bundle-size.js]: :+1:

**Pull request merged**